apps/omstillingsstoenad-ui/src/locales/nn.ts

```diff
@@ -766,6 +766,7 @@ export default {
     'soekbarYtelse.fosterhjemsgodtgjoering': 'Fosterheimsgodtgjersle',
     'soekbarYtelse.omsorgspenger': 'Omsorgspengar',
     'soekbarYtelse.opplaeringspenger': 'Opplæringspengar',
+    'soekbarYtelse.ufoeretrygd': 'Uføretrygd',
     'soekbarYtelse.ufoerepensjon': 'Uførepensjon',
```

Comment:
Brukes `soekbarYtelse.ufoerepensjon` fortsatt noe sted?
Gjelder for alle språklag

Reply:
Ja, den brukes for for søknad om ytelse fra andre
![image](https://github.com/user-attachments/assets/ecfa4f15-48d7-455e-bc3f-35c9f9f4c595)